src/libServer/JSONConversion.cpp

```diff
@@ -134,11 +146,16 @@ const Json::Value JSONConversion::convertDSblocktoJson(const DSBlock& dsblock) {
   ret_header["DifficultyDS"] = dshead.GetDSDifficulty();
   ret_header["GasPrice"] = dshead.GetGasPrice().str();
   ret_header["PoWWinners"] = Json::Value(Json::arrayValue);
+  ret_header["PoWLoosers"] = Json::Value(Json::arrayValue);
 
   for (const auto& dswinner : dshead.GetDSPoWWinners()) {
     ret_header["PoWWinners"].append(static_cast<string>(dswinner.first));
   }
 
+  for (const auto& dslooser : dshead.GetDSRemovePubKeys()) {
+    ret_header["PoWLoosers"].append(static_cast<string>(dslooser));
```

Comment:
maybe not loser since they didn't lose the pow. 
Maybe `MembersChurned`?

@ansnunez Do you have any good suggestion too?

Reply:
> maybe not loser since they didn't lose the pow.
> Maybe `MembersChurned`?
> 
> @ansnunez Do you have any good suggestion too?

“Ejected”?

Reply:
ejected sound good to me